archstuff/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: Inexor <https://discord.gg/Sz4SKXpxRN>
```

Comment:
Please rework this file using the tips given in the old MR #357

Reply:
did it